.dockerignore

```diff
@@ -2,8 +2,10 @@ tmp/
 log/
 .git/
 node_modules/
-spec/
+spec
 .*
+*.md
+*.yaml
+*.yml
```

Comment:
Have you tested this? I would have thought we needed it for https://github.com/mirego/killswitch/blob/main/config/secrets.yml and other config files

Reply:
Good catch, reverting.